fusionfusion/annotationFunction.py

```diff
@@ -126,7 +126,7 @@ def get_junc_info(chr, pos, ref_exon_tb, ens_exon_tb, junction_margin):
 
 
 
-def filterAndAnnotation(inputFilePath, outputFilePath, genome_id, is_grc):
+def filterAndAnnotation(inputFilePath, outputFilePath, genome_id, is_grc, includes_scores=True):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

```suggestion
def filterAndAnnotation(inputFilePath,
                        outputFilePath,
                        genome_id,
                        is_grc,
                        includes_scores=True):
```